index.html

```diff
@@ -231,6 +231,24 @@ <h4 class="txt-lg">Skills</h4>
     </ul>
 
   </section>
+
+  <section class="contact">
+
+    <h1 class="txt-ctct-me">Contact me</h1>
+    <p class="ctct-me">If you have an application you are interested in developing, a feature that you need built or a
+      project that
+      needs coding. I’d love to help with it </p>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the text is aligned to the center for the `p` and `h1` tags as it has been provided in the design template.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:

![Screenshot from 2022-02-15 21-14-37](https://user-images.githubusercontent.com/58692590/154144302-a9ae5e83-8baf-4379-8700-f7529d06578f.png) | ![Screenshot from 2022-02-15 21-14-12](https://user-images.githubusercontent.com/58692590/154144275-b745d7ae-e33e-4451-ac85-f20b2e921c89.png)